graph/ShortestWay/ShortestWay.h

```diff
@@ -0,0 +1,36 @@
+//
+// Created by emyrmak on 7/9/2019.
+//
+#pragma once
+#include <limits>
+#include <vector>
+#include "Vertices.h"
+
+using Matrix = std::vector<std::vector<int>>;
+using intVector = std::vector<int>;
+using boolVector = std::vector<bool>;
```

Comment:
fun fact:  std::vector<bool> has special implementation, different then the default std::vector.